ArduCopter/mode.cpp

```diff
@@ -406,10 +406,8 @@ int32_t Copter::Mode::get_alt_above_ground(void)
 void Copter::Mode::land_run_vertical_control(bool pause_descent)
 {
 #if PRECISION_LANDING == ENABLED
-    AC_PrecLand &precland = copter.precland;
-
     const bool navigating = pos_control->is_active_xy();
-    bool doing_precision_landing = !ap.land_repo_active && precland.target_acquired() && navigating;
+    bool doing_precision_landing = !ap.land_repo_active && copter.precland.target_acquired() && navigating;
```

Comment:
Is the `copter` part needed? I would think yes, but then I don't see how shadowing was happening.